src/table/body-cell/styles.scss

```diff
@@ -131,11 +127,18 @@ $cell-negative-space-vertical: 2px;
 
   &-content {
     box-sizing: border-box;
-  }
-  &:not(.body-cell-wrap) > .body-cell-content {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
+    block-size: 100%;
+    align-content: center;
+
+    &.body-cell-align-top {
+      align-content: baseline;
+    }
+
+    &:not(.body-cell-wrap) {
```

Comment:
The `.body-cell-wrap` only takes effect on the cell-content element so attaching the style to it directly.